portfolio/src/main/webapp/index.html

```diff
@@ -121,11 +121,13 @@
             </tr>
         </table>
     </div>
-    <form action="/data" method="POST">
+    <!-- POST request for comment below -->
+    <form action="/data" method="POST" onsubmit="return emptyBox()">
         <div class="comments">
-            <textarea name="comment-box"></textarea>
+            <textarea name="comment-box" id="comments-box"></textarea>
             <input type="submit"/>
         </div>
+        <!-- Comments displayed below -->
         <div id="display-comments">
```

Comment:
move this out of the `<form>` element since it's not related to the form submission (i.e. the form is here to create a POST request, whereas this div is here for displaying the results of a GET request.

Reply:
oh, i see now!